.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+    push:
+        branches:
+            - master
```

Comment:
```suggestion
            - master
            - beta
      - next
      - '^([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*|x)'
```

Reply:
@niloofar-deriv, as I mentioned in [this](https://github.com/deriv-com/deriv-api-hooks/pull/26#discussion_r1569979549) comment, as far as I know, we only release from the `master` branch. We don't release from any other branch. When needed, it'll be added.